src/index.js

```diff
@@ -8,6 +8,7 @@ import RedBox from './components/RedBox';
 import View from './components/View';
 import Text from './components/Text';
 import TextStyles from './sharedStyles/TextStyles';
+import Symbol from './symbol';
```

Comment:
Sure, that's fine :-) it can conceptually overlap, as long as it doesn't literally overlap.

However, I'd also say that this doesn't tell me what a "symbol" is in this context, and it's not super clear to me from the OP either :-/